tests/test_interval_set.py

```diff
@@ -511,8 +515,51 @@ def test_save_npz():
     Path("ep.npz").unlink()
     Path("ep2.npz").unlink()  
 
+def test_split():
+    np.random.seed(0)
+    start = np.round(np.random.uniform(0, 10))
+    end = np.round(np.random.uniform(90, 100))
+    tmp = np.linspace(start, end, 100)    
+    interval_size = np.round(tmp[1] - tmp[0], 9)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")    
+        ep0 = nap.IntervalSet(tmp[0:-1], tmp[1:])
+    ep = nap.IntervalSet(tmp[0], tmp[-1])
+    ep1 = ep.split(interval_size)
+    np.testing.assert_array_almost_equal(ep0, ep1)
+
+    # Test with a smaller epochs
+    start = np.hstack((tmp[0:-1], np.array([200])))
+    end = np.hstack((tmp[1:], np.array([200+0.9*interval_size])))
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")    
+        ep2 = nap.IntervalSet(start, end)
+
+    ep = nap.IntervalSet([start[0], 200], end[-2:])
+    ep1 = ep.split(interval_size)
+    np.testing.assert_array_almost_equal(ep0, ep1)
+
+    # Empty intervalset
+    ep = nap.IntervalSet([], [])
+    assert len(ep.split(1)) == 0
+
+def test_split_errors():
+    start = [0, 10, 16, 25]
+    end = [5, 15, 20, 40]
+    ep = nap.IntervalSet(start=start, end=end)
 
+    with pytest.raises(IOError) as e:
+        ep.split('a')
+    assert str(e.value) == "Argument interval_size should of type float or int"
```

Comment:
```suggestion
    with pytest.raises(IOError,  match="Argument interval_size should of type float or int"):
        ep.split('a')
```

you can simplify the test if you want